django_fido/management/commands/download_authenticator_metadata.py

```diff
@@ -73,5 +134,9 @@ def handle(self, **options):
             authenticator.metadata_entry = json.dumps(authenticator_data)
             auth_metadata = requests.get(url, params={'token': SETTINGS.metadata_service['access_token']},
                                          timeout=SETTINGS.metadata_service['timeout'])
-            authenticator.detailed_metadata_entry = urlsafe_b64decode(auth_metadata.content).decode()
+            hash = hashlib.new(alg, auth_metadata.content)
```

Comment:
I believe supported algorithm may vary.

Reply:
Only SHA-256, SHA-384 and SHA-512 are specified in JWA specifications and they are in the guaranteed alg for hashlib, so they are always available.

Reply:
Check for supported algs added anyway.